assets/js/base/components/read-more/index.tsx

```diff
@@ -70,7 +140,7 @@ class ReadMore extends Component {
 		}
 	}
 
-	getButton() {
+	getButton(): JSX.Element | undefined {
```

Comment:
Mm, sure though I was just typing the existing situation here. As I'm not sure about the current test coverage, this could potentially have side effects. Though likely people will check for falsy values and not specifically for `undefined` and, even more likely, nobody outside of the component is consuming this function which should be `private`. 

I was just super cautious I guess 😬

Reply:
Yeah also I don't think they will be checking the explicit value of a method that renders markup but worth checking if you think so